app/models/pdc_serialization/datacite.rb

```diff
@@ -224,15 +224,15 @@ def rights_from_work_resource(resource)
         end
 
         def funding_references_from_work_resource(resource)
+          type_ror = ::Datacite::Mapping::FunderIdentifierType::ROR
           resource.funders.map do |funder|
-            award = ::Datacite::Mapping::AwardNumber.new(uri: funder.award_uri, value: funder.award_number)
-            if funder.ror.present?
-              type = ::Datacite::Mapping::FunderIdentifierType::ROR
-              funder_identifier = ::Datacite::Mapping::FunderIdentifier.new(type: type, value: funder.ror)
-              ::Datacite::Mapping::FundingReference.new(name: funder.funder_name, award_number: award, identifier: funder_identifier)
-            else
-              ::Datacite::Mapping::FundingReference.new(name: funder.funder_name, award_number: award)
-            end
+            award = if funder.award_number.present?
+                      ::Datacite::Mapping::AwardNumber.new(uri: funder.award_uri, value: funder.award_number)
+                    end
+            funder_identifier = if funder.ror.present?
+                                  ::Datacite::Mapping::FunderIdentifier.new(type: type_ror, value: funder.ror)
+                                end
+            ::Datacite::Mapping::FundingReference.new(name: funder.funder_name, award_number: award, identifier: funder_identifier)
           end
```

Comment:
The new logic only uses the award information for DataCite serialization if there is a award number indicated.